packages/neo-one-cli/src/commands/startServer.ts

```diff
@@ -12,22 +13,24 @@ export const startServer = (args: CLIArgs) => {
       serverConfig,
       binary,
     }).subscribe({
-      error: (error) => {
+      error: async (error: Error) => {
```

Comment:
Remove from here - this will just log the lines of the server, which is exactly what is running.